.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
   - pipenv install --system && pipenv check
   - cd ..
 script:
+  - bash scripts/db_docs_generator.sh
```

Comment:
could this command be executed in `after_success` section instead?

Reply:
should be fixed, also I changed the commit name for the semantic release

Reply:
also changed for vmaas